GiEnJul/ClientApp/src/components/registerGiver/PrivacyDialog.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Typography,
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+import { FC } from "react";
+import useStyles from "./Styles";
+
+interface IPrivacyDialog {
+  privacyState: boolean;
+  setPrivacyState: (isChecked: boolean) => void;
+  onClose: () => void;
+  open: boolean;
+}
+
+const PrivacyTexts = {
+  headers: [
+    "Behandlingsansvarlig",
+    "Personopplysninger som lagres",
+    "Formål med behandlingen",
+    "Grunnlaget for behandlingen",
+    "Innhenting av personopplysninger",
+    "Utlevering av opplysninger til tredjeparter",
+    "Sletting av personopplysninger",
+    "Rettigheter for den registrerte",
+    "Informasjonssikkerhet",
+  ],
+  text: [
+    "Karianne Munch-Ellingsen er på vegne av Gi en jul behandlingsansvarlig for selskapets behandling av personopplysninger.",
+    "Vi lagrer følgende personopplysninger om våre kunder:navn, adresse, telefonnummer, e-postadresse.",
+    "Vi behandler opplysningene for å kunne gjennomføre prosjektet. Personopplysninger vil bli brukt for å koble givere og familier, informasjonsflyt angående prosjektet og generell kontakt om det er nødvendig.",
+    "Informasjon om navn, telefon, e-postadresse benyttes for å oppfylle avtalen. Grunnlaget for denne behandlingen er personvernforordningens artikkel Art 6 (b).",
+    "Vi lagrer de personopplysningene du har avgitt på våre nettsider.Vi bruker informasjonskapsler/cookies på våre nettsider for å gi deg som besøker siden best kundeopplevelse og service.Lov om elektronisk kommunikasjon krever at vi informerer våre besøkende om bruk av informasjonskapsler (cookies). Les mer om bruk av informasjonskapsler her:",
+    "Vi vil ikke dele, selge, overføre eller på annen måte utlevere personopplysninger til andre, med mindre vi er rettslig forpliktet til det.",
+    "Opplysninger vi har mottatt i forbindelse med ditt kjøp lagres i vårt aktive database i 13 måneder.",
+    "Vi behandler dine personopplysninger i henhold til personopplysningsloven og gjeldende forskrifter. Det gjøres oppmerksom på at du kan kreve innsyn i og flytting av egne personopplysninger, samt kreve retting eller sletting av opplysninger. Det kan klages til Datatilsynet på behandling i strid med reglene.",
+    "Vi sikrer dine personopplysninger ved både fysisk og virtuell adgangs- og tilgangskontroll, samt ved kryptering av sensitive deler av avgitte opplysninger.",
```

Comment:
```suggestion
    "Vi sikrer dine personopplysninger ved best practice for situasjonen",
```

Reply:
This definition is somewhat vague. Personal-info should be encrypted as we learnt earlier. Maybe there should be created a issue on the topic(encrypt personal-data - priority high) and then the Personvernerklæring could let the user know that data is encrypted.